docs/source/big_modeling.mdx

```diff
@@ -203,6 +203,50 @@ This only supports inference of your model, not training. Most of the computatio
 
 </Tip>
 
+## Designing a `device_map`
```

Comment:
I think including the ticks here may have broken the right navbar in the docs?

Reply:
Arg!